data_request_api/stable/utilities/config.py

```diff
@@ -75,9 +78,6 @@ def load_config() -> dict:
                 CONFIG = yaml.safe_load(f)
         except FileNotFoundError:
             pass
-        except yaml.YAMLError as e:
-            print(f"Error parsing config file: {e}")
-            raise
```

Comment:
This was removed since `yaml.safe_load(file)` already captures all YAMLErrors and so this part is only affecting the test coverage.